app/src/main/kotlin/com/wire/android/ui/userprofile/other/bottomsheet/OtherUserBottomSheetState.kt

```diff
@@ -21,21 +25,29 @@ class OtherUserBottomSheetState {
     }
 
     private fun updateBottomSheetContentIfNeeded() {
-        when (bottomSheetContentState) {
-            is BottomSheetContent.Conversation -> toConversation()
-            is BottomSheetContent.ChangeRole -> toChangeRole()
-            null -> {}
+        when (flag) {
+            OtherUserBottomSheetContentFlag.CONVERSATION -> toConversation()
+            OtherUserBottomSheetContentFlag.CHANGE_ROLE -> toChangeRole()
+            else -> {}
```

Comment:
Shouldn't we keep `null` or add `NONE` here? so in case any other option is added to `OtherUserBottomSheetContentFlag` we will not be streaming it down to `{}`? 🤔